src/dune_rules/inline_tests_info.ml

```diff
@@ -119,6 +120,9 @@ module Tests = struct
        and+ backend = field_o "backend" (located Lib_name.decode)
        and+ libraries =
          field "libraries" (repeat (located Lib_name.decode)) ~default:[]
+       and+ link_flags =
+         Ordered_set_lang.Unexpanded.field "link_flags"
+           ~check:(Dune_lang.Syntax.since Stanza.syntax (2, 9))
```

Comment:
This should be in 3.0. 2.9 has already been released.

Given that this field is used to customize the inline tests executable, it should go under the `executable` field.